app/policies/application_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Default Policies
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
```

Comment:
I think we should simply rename this to `resource`. Both mean the same thing, but Pundit calls it `record` in the docs. Actually, record represents a row in a database while resource is more general.